ui/components/app/modals/contract-details-modal/contract-details-modal.js

```diff
@@ -160,9 +159,10 @@ export default function ContractDetailsModal({
                 </Box>
                 <Box marginTop={5} marginRight={5}>
                   <Tooltip position="top" title={t('openInBlockExplorer')}>
-                    <Button
+                    <ButtonIcon
```

Comment:
I know these didn't have aria-labels to begin with but could you add them? It's a required prop of the `ButtonIcon` component and helps with screen readers. Docs here https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-buttonicon--default-story#aria-label